### PR TITLE
Dang/add test spam mitigation

### PR DESCRIPTION
**Pull request by GNaD13**
## Description
Add scripts test spam transaction
Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by mustafapylons** [comment]
@nghuyenthevinh2000 can we test spam mitigation before 9/30 latest?

File: app/ante_gate_test.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	errCode     = uint32(19)
+	errCode     = uint32(19) // spam prevent respone code
```
Comment (mustafapylons) [inline app/ante_gate_test.go:22]: // spam prevention response code is a better comment here.

**Review by mustafapylons — COMMENTED** [review]
Small typo in the comment section, otherwise code is good.

**Review by nghuyenthevinh2000 — APPROVED** [review]

File: app/ante_gate_test.go (hunk 1 above)
Comment (GNaD13) [reply app/ante_gate_test.go]: fixed

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Review by MikeSofaer — APPROVED** [review]

**Pull request merged**